app/models/status_message.rb

```diff
@@ -162,28 +164,30 @@ def address
   end
 
   protected
+
   def presence_of_content
     if text_and_photos_blank?
-      errors[:base] << "Cannot create a StatusMessage without content"
+      errors[:base] << 'Cannot create a StatusMessage without content'
     end
   end
 
   def absence_of_content
     unless text_and_photos_blank?
-      errors[:base] << "Cannot destory a StatusMessage with text and/or photos present"
+      errors[:base] << 'Cannot destory a StatusMessage with text and/or photos present'
     end
   end
 
   def filter_mentions
-    return if self.public? || self.aspects.empty?
+    return if self.public? || aspects.empty?
 
-    author_usr = self.author.try(:owner)
-    aspect_ids = self.aspects.map(&:id)
+    author_usr = author.try(:owner)
+    aspect_ids = aspects.map(&:id)
```

Comment:
Isn't aspects a relation there? Should have an `.ids` method or use `.pluck(:id)`